### PR TITLE
Documentation Working Group

### PR DESCRIPTION
**Pull request by tedsuo**
## Documentation Working Group

The OpenTracing community is growing rapidly, moving from a tight-knit group of early adopters to a larger group of developers. In order to support this transition, the Documentation Working Group is tasked with codifying the existing knowledge base into proper documentation which can be easily accessed via the OpenTracing website.

### Documentation Projects
- Project Overview
- Cross-Language Specification
- Tags
- Language APIs
- OpenTracing CookBook
- Walkthroughs and Tutorials
- FAQ
- Index of Instrumentations
- Guidelines for framework/library maintainers

### Responsibilities
- Schedule and attend Documentation Working Group sessions.
- Maintain community guidelines for contributing documentation changes.
- Setup and maintain documentation infrastructure.
- Produce missing documentation.

## Initial agenda for Working Group sessions
- What tools should we use for writing and publishing documentation?
- How can we make a Search feature for OT instrumentations, tools, etc
- Assign first round of documentation/writing tasks

## Initial Membership
If you are interested in joining this Working Group, please let @tedsuo know or reply below.

### CODE REVIEW

**Comment by itsderek23** [comment]
Would like to help @tedsuo! Thx for kicking this off.

**Comment by pritianka** [comment]
Same here! Thanks for the kick off.

Priyanka Sharma
Advisor, HeavyBit <http://heavybit.com/>
Product Marketing & Partnerships, LightStep <http://lightstep.com/>
Co-founder, WakaTime <http://www.wakatime.com/> - Quantify your coding
www.twitter.com/pritianka
650-796-7125

**Comment by griels** [comment]
Hi I'm interested in shaping the general API and the C/C++ and Python implementations.

**Comment by JStickler** [comment]
I'm interested in contributing.

**Comment by ringerc** [comment]
Cannot currently commit to membership but happy to review changes and comment/submit things. Been working on the Jaeger C++ docs and trying to ask clarifying questions on the opentracing c++ docs.

**Comment by tiffon** [comment]
I'm interested in contributing.

**Comment by tedsuo** [comment]
Awesome! I'll leave this issue open till the end of the week, and begin organizing on Monday.

File: project_organization.md [hunk 1]
```diff
@@ -55,6 +55,27 @@ OTIAB members have the following primary responsibilities:
 - Provide feedback about possible priorities and/or specific proposals from the OTSC
 - Represent OpenTracing’s interests within their own organizations
 
+## Documentation Working Group
+
+The OpenTracing community is growing rapidly, moving from a tight-knit group of early adopters to a larger group of developers. In order to support this transition, the Documentation Working Group is tasked with codifying the existing knowledge base into proper documentation which can be easily accessed via the OpenTracing website.
```
Comment (bhs) [inline project_organization.md]: Seems like the "documentation working group" is something that will always exist (i.e., it's not transitional). This paragraph feels like a great thing for the PR description, but for this actual `.md` file I'd rather just describe what the doc working group **is** without referring to the point in time when this paragraph happened to be added to the `.md` file. E.g., maybe rewrite to be just the following:

```
The Documentation Working Group codifies and maintains the existing knowledge base as  proper documentation which can be easily discovered via the OpenTracing website.
```
?

**Comment by tedsuo** [comment]
I've merged this PR, and the working group is now created! I've invited everyone who responded in this issue to the working group gitter channel, and created a document for brainstorming an agenda. We will discuss meeting formats on gitter and schedule an initial working group session.

Gitter Channel:
https://gitter.im/opentracing/docs

Agenda Proposals:
https://docs.google.com/document/d/1fu5io0cTaBKAy-ykFyiY1EztaKC0Fd7YF9ZB4sB4cpg/

Cheers,
@tedsuo

**Pull request merged**